ckb_syscall_apis.h

```diff
@@ -42,4 +42,18 @@ int ckb_dlopen2(const uint8_t* dep_cell_hash, uint8_t hash_type,
                 size_t* consumed_size);
 void* ckb_dlsym(void* handle, const char* symbol);
 
+typedef struct {
+  uint64_t memory_limit;
+  int8_t* exit_code;
+  uint8_t* content;
+  uint64_t* content_length;
+} spawn_args;
```

Comment:
Suggestion:
```
typedef struct spawn_args_t {
  uint64_t memory_limit;
  int8_t exit_code;
  uint8_t* content;
  uint64_t content_length;
} spawn_args_t;
```
1. use `spawn_args_t`
2. `spawn_args_t` is passed as pointer. The `exit_code` can receive values now.
3. Same for `content_length`